pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -269,6 +271,34 @@ var _ = ginkgo.Describe(dedicatedAdminTestName, func() {
 			err := manageSubscriptions(ctx, namespaceList, h)
 			Expect(err).NotTo(HaveOccurred())
 		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// dedicated-admin user can patch consoles.operator.openshift.io CR
```

Comment:
unneeded comment (prefer `It`, `By,`, or `Context` to inline comments)

Reply:
Thank you for the rebview. I have address this in my latest commit